### PR TITLE
Add kitty and alacritty to list of Linux terminals

### PR DESCRIPTION
**Pull request by madskjeldgaard**
## Purpose and Motivation

This adds to the awesome work made on the LinuxPlatform class in 3.12 where .runInTerminal was improved and had default terminals added to it. The list is missing two (gpu accelerated and) popular terminals: kitty and alacritty and so this is a simple PR that adds those two

## Types of changes

- New feature

## To-do list

- [ ] Code is tested
- [ ] All tests are passing
- [ ] Updated documentation
- [x ] This PR is ready for review

### CODE REVIEW

File: SCClassLibrary/Platform/linux/LinuxPlatform.sc [hunk 1]
```diff
@@ -49,11 +49,13 @@ LinuxPlatform : UnixPlatform {
 			"rxvt -T % -e %",
 			"terminator -T % -e %",
 			"xterm -T % -e %",
+            "kitty -T % -e %",
+            "alacritty -t % -e %",
```
Comment (dyfer) [inline SCClassLibrary/Platform/linux/LinuxPlatform.sc:53]: indentation seems different than in surrounding code

File: SCClassLibrary/Platform/linux/LinuxPlatform.sc [hunk 2]
```diff
@@ -49,11 +49,13 @@ LinuxPlatform : UnixPlatform {
...
 			// DE-specific terminals last: avoid problems if not on GNOME or KDE but term installed
 			"xfce4-terminal -T % -e %",
 			"mate-terminal -t % -e %",
 			"gnome-terminal -t % -- %",
-			"konsole --title % -e %",
+			"konsole --title % -e %"
```
Comment (dyfer) [inline SCClassLibrary/Platform/linux/LinuxPlatform.sc:58]: is this change needed?

**Review by dyfer — COMMENTED** [review]
Thanks! I have some minor comments

File: SCClassLibrary/Platform/linux/LinuxPlatform.sc (hunk 2 above)
Comment (madskjeldgaard) [reply SCClassLibrary/Platform/linux/LinuxPlatform.sc:58]: Hmm, perhaps not - I'll reinstate the comma

**Comment by madskjeldgaard** [comment]
should be good now @dyfer  thanks

**Comment by dyfer** [comment]
I think the indentation is still off - in the class library [we use tabs for indentation](https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#rule-use-tabs-for-indentation)

**Comment by madskjeldgaard** [comment]
> I think the indentation is still off - in the class library [we use tabs for indentation](https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#rule-use-tabs-for-indentation)

Sorry, had never noticed this but I think I found a way to change this setting in my editor. Hope it's better now :)

**Comment by dyfer** [comment]
Thanks for the changes! The tabs are good now.
Sorry to be a pain about this, but there are also spaces after the commas (before the newline), it would be great to remove those as well, I noticed them by looking at [this part of the diff](https://github.com/supercollider/supercollider/pull/5548/files#diff-2c0665171d7933ed14a0d8598c8ad3583d44599dde44592d92060e410cb807a9L51-R51)

As for the functionality, I'll try to test in the next few days so that this can be merged.

**Comment by madskjeldgaard** [comment]
Sorry about the whitespace. Should be removed now.


-- 
Best regards, Mads Kjeldgaard

**Review by dyfer — APPROVED** [review]
Works as advertised, thanks!

**Comment by madskjeldgaard** [comment]
Hi @dyfer has this been included in a release yet?

**Comment by dyfer** [comment]
Hi @madskjeldgaard, sorry, I don't think so. It was off my radar and I forgot to cherry-pick it. But we will likely be releasing 3.13 soon-ish, it will be included there. Is that okay?

**Comment by madskjeldgaard** [comment]
Yeah that's great. Thanks Marcin !
